todo_rxbloc_app/lib/feature_todos_list/bloc/todos_list_manage_bloc.dart

```diff
@@ -44,6 +54,28 @@ class TodosListManageBloc extends $TodosListManageBloc {
         .bind(_$deleteTodoEvent)
         .addTo(_compositeSubscription);
   }
+  @override
+  ConnectableStream<void> _mapToDeleteMarkedTodosState() => _$deleteMarkedEvent
+      .withLatestFrom<List<TodoEntity>, List<TodoEntity>>(
+          todosList, (_, todos) => todos)
+      .switchMap((value) => todoService
+          .deleteMarkedTodos(
+              value.where((element) => element.complete == true).toList())
+          .asResultStream()
+          .setResultStateHandler(this)
+          .whereSuccess())
+      .publish();
+
+  @override
+  ConnectableStream<List<TodoEntity>> _mapToTodosListState() =>
+      _$getTodosListEvent
+          .startWith(null)
+          .switchMap((value) => todoService
+              .getTodos(FilterEnum.showAll)
+              .asResultStream()
+              .setResultStateHandler(this)
```

Comment:
.setResultStateHandler(this) and .whereSuccess() must be placed outside of the switchMap operator